specs/electra/beacon-chain.md

```diff
@@ -155,7 +153,7 @@ The following values are (non-configurable) constants used throughout the specif
 
 | Name | Value | Unit |
 | - | - | :-: |
-| `PENDING_BALANCE_DEPOSITS_LIMIT` | `uint64(2**27)` (= 134,217,728) | pending balance deposits |
+| `PENDING_DEPOSITS_LIMIT` | `uint64(2**27)` (= 134,217,728) | pending deposits |
```

Comment:
2^32 would be the theoretical limit with infinite gas fee and a single block that fills the entire deposit tree